sycl/source/detail/usm/usm_dispatch.cpp

```diff
@@ -43,7 +45,35 @@ USMDispatcher::USMDispatcher(cl_platform_id platform) {
                   pfn_clSetKernelArgMemPointerINTEL &&
                   pfn_clEnqueueMemsetINTEL && pfn_clEnqueueMemcpyINTEL);
     mEmulator.reset(new CLUSM());
+
+    if (mEmulated) {
+      // See if every device in this context supports
+      // CL_DEVICE_SVM_FINE_GRAIN_BUFFER
+      // If not, disable USM
+
+      if (CL_TARGET_OPENCL_VERSION >= 200) {
+        bool AnybodyNotSupportSVM = false;
+        for (const auto &D : DeviceIds) {
+          cl_device_svm_capabilities caps;
+          cl_int Error = clGetDeviceInfo(
+              pi::cast<cl_device_id>(D), CL_DEVICE_SVM_CAPABILITIES,
+              sizeof(cl_device_svm_capabilities), &caps, nullptr);
+          AnybodyNotSupportSVM = (Error != CL_SUCCESS) ||
```

Comment:
Shouldn't it be
AnybodyNotSupportSVM &=

Reply:
Updated.  I think it should be |=